templates/post.html

```diff
@@ -3,6 +3,8 @@
         <div class="col-lg-10 col-md-12 mx-auto">
             <h1 class="title">$title$</h1>
             <div class="post-context">
+              <span class="author">$author$</span>
+              <br/>
```

Comment:
I would try to avoid having explicit `<br>` and used either `display: block` or `div` insetead of span. Brs can make it unnecessary difficult to style responsive versions for mobile and tablet.